File: Örnek-6.5/Matlab/kenaruzun.m
a=input('Birinci kenar uzunluğunu giriniz (cm): ');
b=input('İkinci kenar uzunluğunu giriniz (cm): ');
aci=input('Arasındaki açı değerini girniniz (derece): ');

c=sqrt(a^2+b^2-2*a*b*cosd(aci));

fprintf('C kenarın uzunluğu: %0.3f',c);

File: Örnek-6.5/Python/kenaruzun.py
import math

a=float(input('Birinci kenar uzunluğunu giriniz (cm): '))
b=float(input('İkinci kenar uzunluğunu giriniz (cm): '))
açı=float(input('Arasındaki açıyı giriniz (derece): '))

c=math.sqrt(a*a+b*b-2*a*b*math.cos(math.radians(açı)))

print('C kenarın uzunluğu: %0.3f'%(c))